homework.md

```diff
@@ -44,70 +44,73 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+Javascript is a scripting language that enables you to create dynamically updating content, control multimedia, animate images. It's a dynamic programming language that's used for web development, in web applications, for game development, and lots more.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+GitHub is a website for developers and programmers to collaboratively work on code.
+
+The primary benefit of GitHub is its version control system, which allows for seamless collaboration without compromising the integrity of the original project. 
+
+It's a great place to host your portfolio whilst demonstrating the ability to use and understand it. 
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You make a copy of it on your own remote repo branch, so you can freely experiment with changes without affecting the original project.
```

Comment:
Excellent. Just more details, to help cement your understanding

It copies that entire branch - whether it's `master` or another branch, including all of it previous commit (hashes)

Reply:
Though it gives you a separate remote repo origin, such as dharmaguadeloupe/GitHomeworkFixErrors

Reply:
Yes I figured that put panicking with my first project when the history commits page I was looking at was for the original repo, not mine